SU2_CFD/src/integration_time.cpp

```diff
@@ -44,7 +44,8 @@ void CMultiGridIntegration::MultiGrid_Iteration(CGeometry ***geometry,
                                                 unsigned short RunTime_EqSystem,
                                                 unsigned long Iteration,
                                                 unsigned short iZone) {
-  unsigned short FinestMesh, iMGLevel;
+	
+	unsigned short FinestMesh, iMGLevel;
```

Comment:
Please double check 2 spaces vs tabs, as Tim suggested. If you need any help setting this up in Xcode, just let me know (it's in preferences, and you can use ctrl+i to auto indent highlighted code). You can submit more commits straight to this branch after cleaning it up.